set_avl.cc

```diff
@@ -7,7 +7,23 @@
 // key를 root로 하는 subtree에서 최솟값을 갖는 node의 값과 depth를 출력
 void SetAVL::Minimum(const int key)
 {
-
+    NodeAVL* node = root_;
+    // key를 root로 하는 node찾기
+    while(node != nullptr)
+    {
+        if(key == node->GetNum())
+             break;
+        else if(key < node->GetNum())
+            node = node->GetLeft();
+        else
+            node = node->GetRight()
+    }
+    // subtree에서 최솟값을 갖는 node찾기
+    while(node !== nullptr && node->GetLeft() !== nullptr)
+    {
+        node = node->GetLeft();
+    }
+    std::cout << "key : " << node->GetNum() << ", depth : " << Find(key);
```

Comment:
요 부분에서 아예 출력되는 것이 맞나요? 맞다면

<img width="811" alt="image" src="https://github.com/OpenSourceProgramming/INHA_OSAP_004_Froyo/assets/69078056/19c91528-b1cb-43c7-a125-b4c1a4dd9307">
<img width="448" alt="image" src="https://github.com/OpenSourceProgramming/INHA_OSAP_004_Froyo/assets/69078056/f7bf6b46-f87e-474d-8a19-36fe62c51b7b">

문제 예시를 보면 이런식으로 출력된다고 되어있어서 
```suggestion
    std::cout << node->GetNum() << " " << Find(key);
```

요런식으로 출력 모양을 수정해야될 것 같습니당..!

Reply:
네 이게 맞네요. 그리고 Find(key)에서 key 대신 node->GetNum()으로 수정하면 될 것 같습니다. (최대 key를 갖는 노드의 depth를 출력해야 해서 그렇습니다)

Reply:
두 분이 말씀해주신 부분 모두 수정했습니다!
다시 확인 부탁드릴게요!